Library/Homebrew/formulary.rb

```diff
@@ -268,15 +303,19 @@ class FromUrlLoader < FormulaLoader
     def initialize(url)
       @url = url
       uri = URI(url)
-      formula = File.basename(uri.path, ".rb")
+      formula = if File.extname(uri.path) == ".json"
```

Comment:
These sorts of changes make me think this should be a two-step process that's perhaps outside of Formulary:
- if you `brew install $JSON` then download the relevant bottles
- turn this into `brew install $LOCAL_BOTTLE_FILE` for each of them working through the dependency tree

This would avoid having any need somehow read a formula from a JSON file. Thoughts?